prow/spyglass/viewers/buildlog/buildlog_template.go

```diff
@@ -41,6 +41,7 @@ var buildLogTemplateText = `<style>
 .skipped {
   display: none;
 }
+.shown {}
```

Comment:
Is this supposed to have some content? It's used, but empty.

Reply:
I just left it in as a placeholder. Not really necessary though

Reply:
imo you should remove that then